libwayshot/src/lib.rs

```diff
@@ -366,63 +336,35 @@ impl WayshotConnection {
             tracing::error!("You can send a feature request for the above format to the mailing list for wayshot over at https://sr.ht/~shinyzenith/wayshot.");
             return Err(Error::NoSupportedBufferFormat);
         };
-        Ok(FrameCopy {
-            frame_format,
-            frame_color_type,
-            frame_mmap,
-            transform,
-        })
+        Ok((
+            FrameCopy {
+                frame_format,
+                frame_color_type,
+                frame_mmap,
+                transform: output_info.transform,
+                position: (
+                    output_info.dimensions.x as i64,
+                    output_info.dimensions.y as i64,
+                ),
+            },
+            frame_guard,
+        ))
     }
 
-    fn create_frame_copy(
```

Comment:
This ties in with comment: https://github.com/waycrate/wayshot/pull/71#issuecomment-1782375785

Should we really opt in for full screencopys always?

Reply:
Will make it still support not doing full screencopies. Elsewhere the comment was made that this is still used for screen recording, where it definitely makes sense and I had not kept that in mind.